libs/hdf-converters/index.ts

```diff
@@ -33,6 +33,7 @@ export * from './src/scoutsuite-mapper';
 export * from './src/snyk-mapper';
 export * from './src/sonarqube-mapper';
 export * from './src/splunk-mapper';
+export * from './src/trufflehog-mapper';
```

Comment:
add the mapper to the list of mappers supported in the supported formats tooltip in the load modal within heimdall